code/game/sound.dm

```diff
@@ -1,3 +1,28 @@
+
+///Default override for echo
+/sound
```

Comment:
I'm still confused, is this specifically the sound that makes environments possible again?, could you just clarify that

i.e 
///Default override sound for echo, prevents environments impacting sounds that don't need to apply them.

Reply:
it's overwriting the default value for echo on all sounds

Reply:
I understand what it's doing, but I do not understand completely how it makes sound environments possible without reverb

Reply:
So the Room effect on the environment is what causes the reverb. Using echo, you can mitigate or remove that effect on a specific sound by changing the room value for each frequency level.

Reply:
Here is all of the documentation I can find on this stuff. It's pretty ancient.

https://documentation.help/FMOD-Ex/documentation.pdf page 144
 https://flylib.com/books/en/3.375.1.40/1/

Reply:
thanks